backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/utils/DeviceUtil.kt

```diff
@@ -22,9 +22,23 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.platform.LocalConfiguration
 
 @Composable
-internal fun isTablet(): Boolean {
+internal fun isSmallTablet(): Boolean {
     val configuration = LocalConfiguration.current
     return configuration.screenWidthDp > SMALL_TABLET_WIDTH
 }
 
+@Composable
+internal fun isTablet(): Boolean {
+    val configuration = LocalConfiguration.current
+    return configuration.screenWidthDp > TABLET_WIDTH
+}
+
+@Composable
+internal fun isDesktop(): Boolean {
+    val configuration = LocalConfiguration.current
+    return configuration.screenWidthDp > TABLET_WIDTH
+}
+
 const val SMALL_TABLET_WIDTH = 513
+const val TABLET_WIDTH = 769
+const val DESKTOP_WIDTH = 1025
```

Comment:
More magic numbers here

Reply:
I'd also say they're fine as long as we have them in a central place and don't redefine what these are with each components